src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1117,6 +1117,7 @@ protected override void GetItem(string path)
 
             try
             {
+#if !UNIX
                 bool retrieveStreams = false;
                 FileSystemProviderGetItemDynamicParameters dynamicParameters = null;
```

Comment:
I think the fix should be to make the dynamic parameters related to `Stream` operation unavailable in FileSystemProvider. Otherwise, it's broken user experience as I can use the `-Stream` parameter but it doesn't behave as expected.